### PR TITLE
Evaluation test for CI pipeline

### PR DESCRIPTION
**Pull request by aidanjgriffiths**
Hey @baradghimire,
Have a look at this test I put together for the evaluation workflow. Let me know how you go as I might need to add to this request as I update the documentation of setting up the project environment. Cheers

### CODE REVIEW

**Review by baradghimire — COMMENTED** [review]
Everything looks good to me. Can you look into the value for `dataset_path` as it might need to be changed for the CI server?

File: speaker_verification/model_evaluation.py [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
 def run_VCSK_Corpus_data(speaker_1, speaker_2, to_csv):
     np.random.seed(123)
     seed(123)
 
     model = DeepSpeakerModel()
-    model.rescnn.load_weights(model_path, by_name=True)
+    model.rescnn.load_weights(MODEL_PATH, by_name=True)
     dataset_path = (
-        "../../datasets/VCTK_Corpus_Fileshare/VCTK_Corpus/wav48_silence_trimmed"
+        "/home/aidan/dev/machine_learning/datasets/VCTK_Corpus_Fileshare/VCTK_Corpus/wav48_silence_trimmed/"
```
Comment (baradghimire) [inline speaker_verification/model_evaluation.py:23]: Can you look into this value for `dataset_path` as it might need to be changed for the CI server?

Comment (aidanjgriffiths) [reply speaker_verification/model_evaluation.py]: This code isn't run in the pytest tests, I've made a seperate `tests` directory with the test using `flac` files within the repository. This won't be an issue but probably should be refactored out of the module as it was primary goal was to create the `csv` files from the first iteration.

File: speaker_verification/model_evaluation.py [hunk 2]
```diff
@@ -2,26 +2,28 @@
...
 
-model_path = join(abspath(dirname(__file__)), "models", "ResCNN_triplet_training_checkpoint_265.h5")
 
-def run_VCSK_Corpus_data(speaker_1, speaker_2, to_csv):
+MODEL_PATH = join(
+    abspath(dirname(__file__)), "models", "ResCNN_triplet_training_checkpoint_265.h5"
+)
+
+
+def run_VCSK_Corpus_data(speaker_1, speaker_2, to_csv, dataset_path):
```
Comment (aidanjgriffiths) [inline speaker_verification/model_evaluation.py:22]: @baradghimire I've just added the `dataset_path` as a quick fix to the local path.

**Pull request merged**